src/manifest.jl

```diff
@@ -133,13 +137,14 @@ function Manifest(raw::Dict)::Manifest
         uuid = nothing
         deps = nothing
         try
-            uuid           = read_field("uuid",          nothing, info, safe_uuid)
-            entry.pinned   = read_pinned(get(info, "pinned", nothing))
-            entry.version  = read_field("version",       nothing, info, safe_version)
-            entry.path     = read_field("path",          nothing, info, safe_path)
-            entry.repo.url = read_field("repo-url",      nothing, info, identity)
-            entry.repo.rev = read_field("repo-rev",      nothing, info, identity)
-            entry.tree_hash = read_field("git-tree-sha1", nothing, info, safe_SHA1)
+            entry.pinned      = read_pinned(get(info, "pinned", nothing))
+            uuid              = read_field("uuid",          nothing, info, safe_uuid)
+            entry.version     = read_field("version",       nothing, info, safe_version)
+            entry.path        = read_field("path",          nothing, info, safe_path)
+            entry.repo.remote = read_remote(get(info, "repo-remote", nothing))
```

Comment:
What is this field?

Reply:
Ah its just a new name for `repo.url`.

Reply:
`repo.source` was `repo.url`. And `repo.remote` means that the source is not on the local filesystem

Reply:
I'll rework this so it does not introduce manifest changes